### PR TITLE
Make FuretUI work with MsSQL

### PR DESCRIPTION
**Pull request by GohuHQ**
```
sqlalchemy.exc.OperationalError: (pymssql.OperationalError) (169, b'A column has been specified more than once in the order by list. Columns in the order by list must be unique.DB-Lib error message 20018, severity 15:\nGeneral SQL Server error: Check messages from the SQL Server\n')
[SQL: SELECT furetui_menu_sub.id AS furetui_menu_sub_id, furetui_menu_sub.sequence AS furetui_menu_sub_sequence, furetui_menu_sub.label AS furetui_menu_sub_label, furetui_menu_sub.properties AS furetui_menu_sub_properties, furetui_menu_sub.login_state AS furetui_menu_sub_login_state, furetui_menu_sub.parent_id AS furetui_menu_sub_parent_id 
FROM furetui_menu_sub 
WHERE furetui_menu_sub.parent_id = %(param_1)s AND furetui_menu_sub.login_state IN (%(login_state_1)s, %(login_state_2)s) ORDER BY furetui_menu_sub.sequence, furetui_menu_sub.sequence]
[parameters: {'param_1': 1, 'login_state_1': 'unlogged', 'login_state_2': 'both'}]
```

### CODE REVIEW

File: anyblok_furetui/furetui/menus.py [hunk 1]
```diff
@@ -124,7 +124,6 @@ def get_for(cls, parent, authenticated_userid):
         # TODO raise if MENU_TYPE is None
         query = cls.query()
         query = query.filter(cls.parent == parent)
-        query = query.order_by(cls.sequence)
```
Comment (jssuzanne) [inline anyblok_furetui/furetui/menus.py]: what is the pb, the order by is important here

Comment (GohuHQ) [reply anyblok_furetui/furetui/menus.py]: The order by is done twice. If you inspect the file you can find it on https://github.com/AnyBlok/anyblok_furetui/blob/92099e27f1a9a90fa089c0162f57cc5108f1c737/anyblok_furetui/furetui/menus.py#L130.
And this is a problem for MsSQL be cause it produces a query with `order by sequence, sequence`

**Comment by GohuHQ** [comment]
@jssuzanne J'ai supprimé la compatibilité avec python 3.4 et 3.5.

File: anyblok_furetui/furetui/menus.py (hunk 1 above)
Comment (jssuzanne) [reply anyblok_furetui/furetui/menus.py]: we must found why, and let the order by

**Pull request merged**